src/modules/reporting/components/reporting-dispute/reporting-dispute.jsx

```diff
@@ -15,7 +16,9 @@ export default class ReportingDispute extends Component {
     history: PropTypes.object.isRequired,
     doesUserHaveRep: PropTypes.bool.isRequired,
     markets: PropTypes.array.isRequired,
+    upcomingMarkets: PropTypes.array.isRequired,
     marketsCount: PropTypes.number.isRequired,
+    upcomingMarketsCount: PropTypes.number.isRequired,
```

Comment:
Why use the extra prop for length vs length checking within this file?

Reply:
following existing convention